build_runner/test/build_script_generate/experiments_test.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@Tags(['experiments'])
+@TestOn('vm')
+import 'package:build_runner/build_script_generate.dart';
+import 'package:logging/logging.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('build scripts can use experiments', () async {
```

Comment:
yet another good reason for us to have some kind of fake experiment...

Reply:
yeah it would be better, but at least pinning to a specific SDK here should make this be valid for quite some time.